components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodManager.kt

```diff
@@ -59,6 +63,35 @@ class GitpodManager : Disposable {
         lifetime.terminate()
     }
 
+    private val memProfileMode = System.getenv("JB_MEM_PROFILE").toBoolean()
+
+    init {
+        val memProfileJob = GlobalScope.launch {
+            if (application.isHeadlessEnvironment || !memProfileMode) {
+                return@launch
+            }
+            Files.newBufferedWriter(Paths.get("/tmp/jb_mem_profile.log")).use { writer ->
```

Comment:
`/tmp` folder is calc as memory for our workspace. If we keet appending content, mem will keep increase, once user open workspace for a long time, it'll be a bad experience. It's more terrible than process mem increase (OOM can kill it)

Reply:
yeah, overwriting will be better 👍 

I actually would like to move it to prometheus reporting instead, i.e. look up it on supervisor metrics endpoint

Should I do it in this PR?

Reply:
- If history is not very useful, we can replace all content
- Or make this log rotate delete?

Reply:
it was not useful to me, but in grafana dashboard we will be able to see history too.

Reply:
If you consider this feature as normal users not reachable, we can go with it👍

Reply:
`prometheus` is good place 👍

Reply:
Good for me